daemon/cluster/convert/network.go

```diff
@@ -8,6 +8,7 @@ import (
 	types "github.com/docker/docker/api/types/swarm"
 	netconst "github.com/docker/libnetwork/datastore"
 	swarmapi "github.com/docker/swarmkit/api"
+	allocator "github.com/docker/swarmkit/manager/allocator"
```

Comment:
Importing the package as `allocator` is redundant.

I'm not sure how I feel about importing `allocator` in the engine. To me, this package is internal to swarmkit. It might be better to copy the function.